python/ray/tests/test_object_spilling.py

```diff
@@ -316,6 +342,45 @@ def test_spill_objects_automatically(object_spilling_config, shutdown_only):
     assert_no_thrashing(address["redis_address"])
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_unstable_spill_objects_automatically(unstable_spilling_config, shutdown_only):
```

Comment:
I'd like to merge this one with `test_spill_objects_automatically`, but I failed. I you have some idea, please let me know @rkooo567